sys/net/application_layer/dhcpv6/client.c

```diff
@@ -208,7 +206,20 @@ static inline bool _is_tid(dhcpv6_msg_t *msg)
 
 static inline uint32_t _now_cs(void)
 {
+#if IS_USED(MODULE_ZTIMER)
+    return (uint32_t)(ztimer_now(ZTIMER_MSEC) / US_PER_CS);
```

Comment:
reads like this should use `MS_PER_CS`
```suggestion
    return (uint32_t)(ztimer_now(ZTIMER_MSEC) / MS_PER_CS);
```